airbyte-webapp/src/views/Connector/ServiceForm/ServiceForm.tsx

```diff
@@ -174,10 +174,12 @@ const ServiceForm: React.FC<ServiceFormProps> = (props) => {
   const uiOverrides = useMemo(
     () => ({
       name: {
-        component: (property: FormBaseItem) => <ConnectorNameControl property={property} formType={formType} />,
+        component: (property: FormBaseItem, componentProps: FormComponentOverrideProps) => (
```

Comment:
I'd ideally prefer if we wouldn't need that `componentProps` hack here, and instead have the `disabled` property calculated in this component, but given that we need the `isSubmitting` from the formik state, which only exists a level lower, this is unfortunately not easily achievable atm :-/ But something we might want to keep in mind for later refactorings.